docs/tools/tools_intro.md

```diff
@@ -43,38 +43,10 @@ The Mbed Online Compiler is our in-house IDE and should be familiar to anyone wh
 | Arm v7A       | `Arm Compiler 6.11` |
 | Arm v8M       | `Arm Compiler 6.11` |
 
-For more information, please see the [Online Compiler page](developing-mbed-online-compiler.html).
+<span class="notes">**Note:** Arm Compiler 6 is the default Arm Compiler for Mbed OS development. Most Mbed OS platforms are already compatible with Arm Compiler 6. Some existing targets still supporting Arm Compiler 5 will be migrated to Arm Compiler 6 in the future. Please be aware that you must use Arm Compiler 6 for future development because Arm Compiler 5 support will be deprecated in September 2019.</span>
```

Comment:
Are sure about that date? I was told to not put any dates in the warning that the tools will be printing.

Reply:
@AnotherButler - What do you think?

Reply:
That was a requested specifically by Product this afternoon.

Reply:
@AnotherButler @bridadan - I think this needs to be clarified between Product and TAM as I hear different things from them. If we fail to reach an agreement we should go back to the original message I added.

Reply:
I'll try to get it sorted

Reply:
Mohit says September is necessary to reassure users we aren't violating our "six months warning" deprecation policy.